cmake/std/atdm/utils/set_build_options.sh

```diff
@@ -155,3 +174,11 @@ fi
 # 'cuda' or 'pthread', then that will be selected for the Kokkos backend.
 # Otherwise, if one fo these are not selected and 'openmpi' is present in the
 # build name, then the default Kokkos backend will become 'openmp'!
+
+# Set 'static' or 'shared'
+ATDM_CONFIG_SHARED_LIBS=OFF
```

Comment:
Looks like this line may not be needed given line 20 above.

Reply:
Thank is true.  Just being more explicit.  I guess the better approach would be to unset those vars at the top of the file and then set the default just before the argument parsing.  Would that be more clear?

Reply:
Whichever you prefer is fine.  Just noticed the duplication and worried about some point in the future where one line gets changed but not  the other, etc.